react/week4/src/components/SearchForm.js

```diff
@@ -0,0 +1,26 @@
+import React,{useContext} from 'react';
+import{UserContext} from './FetchUser';
+
+const SearchForm = () => {
+
+const{query,setQuery} = useContext(UserContext);
+
+const UpdateUser=(e) =>{
+    e.preventDefault();
+    setQuery(e.target.value)
+}
+
+
+    return(
+        <div>
+        <form className = 'search-form'>
+            <input className = 'search-bar'
+            type ='text' 
+            value={query || ''}
```

Comment:
if you initialize `query` with `""`, you don't need this anymore.
```suggestion
            value={query}
```